db/schema.rb

```diff
@@ -11,22 +11,11 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20190912112625) do
+ActiveRecord::Schema.define(version: 20190805152649) do
```

Comment:
Looks like a schema update snuck into your PR. Could you drop this change from the PR to be in the safe side?

Reply:
It had to happen one last time didn't it! I've reverted the change now.